index.html

```diff
@@ -20,7 +20,19 @@
 			</div>
 		</header>
 		<main id="main"></main>
-		<footer></footer>
+		<footer>
+			<div class="footer">
```

Comment:
Here would be better to put the class footer directly in the tag footer: `<footer class="footer">` and not creating a div

Reply:
Removed div, used selector footer instead